tests/models/test_cnn_lstm.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+import unittest
+
+import torch
+import random
+import os
+import numpy as np
+
+from pythia.models.cnn_lstm import CNNLSTM
+from pythia.common.registry import registry
+from pythia.common.sample import Sample, SampleList
+from pythia.utils.configuration import ConfigNode, Configuration
+from pythia.utils.general import get_pythia_root
+
+
+class TestModelCNNLSTM(unittest.TestCase):
+    def setUp(self):
+        torch.manual_seed(1234)
+        registry.register("clevr_text_vocab_size", 80)
+        registry.register("clevr_num_final_outputs", 32)
+        config_path = os.path.join(
+            get_pythia_root(), "..", "configs", "vqa", "clevr", "cnn_lstm.yml"
```

Comment:
Maybe worth to implement method `get_pythia_related_path("../configs/vqa/clevr/cnn_lstm.yml")` to make life easier.